advanced-integration/package.json

```diff
@@ -6,7 +6,7 @@
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js",
+    "start": "nodemon server.js",
```

Comment:
question - why do we need to change it to nodemon?

Reply:
Reverted back